packages/@aws-cdk/aws-codebuild/lib/project.ts

```diff
@@ -1023,7 +1060,7 @@ export interface IBuildImage {
  *
  * You can also specify a custom image using one of the static methods:
  *
- * - LinuxBuildImage.fromDockerHub(image)
+ * - LinuxBuildImage.fromDockerRegistry(image[, secretsManagerCredential])
```

Comment:
@skinny85 @SoManyHs @rix0rrr do you think it makes sense to merge this API with the one we have for ECS?

Reply:
In that case, we'd probably need some generic modeling of Docker images, using `@aws-cdk/docker` or something.

Reply:
Discussed offline. We decided not create a general Docker image API. Given that, I believe this code is fine - what do you think @eladb ?